pkg/build/webhook/generic/generic_test.go

```diff
@@ -28,7 +28,7 @@ func GivenRequest(method string) *http.Request {
 }
 
 func GivenRequestWithPayload(t *testing.T, filename string) *http.Request {
-	data, err := ioutil.ReadFile("fixtures/" + filename)
+	data, err := ioutil.ReadFile("../../../../test/fixtures/webhook/generic/" + filename)
```

Comment:
@liggitt it was suggested by @smarterclayton to put all the fixtures from the /pkg subdirs into /test/fixtures to have them all on one place to avoid dups. which make sense.
I don't disagree that it's kinda fragile but we have already been doing that https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/projectstatus_test.go#L226